extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -881,8 +881,7 @@ protected function templateExists( $name ) {
 	 * @see SearchControllerTest::testIsCorporateWiki
 	 */
 	protected function isCorporateWiki() {
-		// TODO: refactor to avoid using wgEnableWikiaHomePageExt
-		return !empty( $this->wg->EnableWikiaHomePageExt );
+		return WikiaPageType::isCorporatePage();
```

Comment:
we could kill this method wrapper here and use `WikiaPageType::isCorporatePage` directly

Reply:
I'm going to review all uses of `WikiaPageType::isCorporatePage` and eventually get rid of it. Search is the only place that has a different functionality on www.wikia.com.